extensions/guacamole-auth-radius/src/main/resources/config/radiusConfig.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Config block which registers Radius-specific field types.
+ */
+angular.module('guacRadius').config(['formServiceProvider',
+        function guacRadiusConfig(formServiceProvider) {
+
+    // Define field for the challenge from the RADIUS service
+    formServiceProvider.registerFieldType('GUAC_RADIUS_CHALLENGE_RESPONSE', {
+        module      : 'guacRadius',
+        controller  : 'radiusResponseController',
+        templateUrl : 'app/ext/radius/templates/radiusResponseField.html'
+    });
+
+    // Define the hidden field for the RADIUS state
+    formServiceProvider.registerFieldType('GUAC_RADIUS_STATE', {
+        module      : 'guacRadius',
+        controller  : 'radiusStateController',
+        template    : '<input class="ng-hide" type=hidden ng-model="model" />'
```

Comment:
Why is this template not defined in a separate file, like you do with `GUAC_RADIUS_CHALLENGE_RESPONSE`?

If you're going to hide this with CSS, I'd suggest using a semantic classname rather than abusing `ng-hide`, but explicit hiding of any sort is unnecessary here: `hidden` input fields are hidden.